<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FertDialogClass</class>
 <widget class="QDialog" name="FertDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Fertiliser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/pic/fert16.png</normaloff>:/pic/fert16.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxType">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Unit:</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxUnit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxUnit">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">dt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">m3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">l</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">t</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">kg</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Contents kg/unit (use negative figures e.g.in case of CaO/Humus withdrawal)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string notr="true">N:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxN</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxN">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string notr="true">P2O5:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxP2o5</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxP2o5">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string notr="true">K2O:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxK2o</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxK2o">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string notr="true">S:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxS</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxS">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string notr="true">MgO:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxMgo</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMgo">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string notr="true">CaO:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxCao</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxCao">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.990000000000009</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string notr="true">Humus:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="dSBHumus">
        <property name="suffix">
         <string notr="true"> kg</string>
        </property>
        <property name="minimum">
         <double>-999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FertDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FertDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
